app/resources/main.qml

```diff
@@ -1,7 +1,7 @@
 import QtQuick 2.1
-import QtQuick.Window 2.0
+import harbour.kaidan 1.0
```

Comment:
Harbour is just a namespace. This is a Jolla convention and doesn't harm the desktop version. If we get in trouble with this in the future, we can rework the naming...